pettingzoo/utils/wrappers/capture_stdout.py

```diff
@@ -1,23 +1,25 @@
 from pettingzoo.utils.capture_stdout import capture_stdout
+from pettingzoo.utils.env import AECEnv
 from pettingzoo.utils.wrappers.base import BaseWrapper
 
 
 class CaptureStdoutWrapper(BaseWrapper):
     """Takes an environment which prints to terminal, and gives it an `ansi` render mode where it captures the terminal output and returns it as a string instead."""
 
-    def __init__(self, env):
+    def __init__(self, env: AECEnv) -> None:
+        assert hasattr(env, "render_mode"), f"Environment {env} has no render_mode."
         assert (
-            env.render_mode == "human"
-        ), f"CaptureStdoutWrapper works only with human rendering mode, but found {env.render_mode} instead."
+            env.render_mode == "human"  # type: ignore
```

Comment:
Instead of ignoring type issues, add explicit assertions

Reply:
There is an assertion in the line before, but pyright still thinks that `env.render_mode` might not be present.

![image](https://github.com/Farama-Foundation/PettingZoo/assets/46611988/da50b08b-9acf-400f-ae4f-baf34c4abcc7)

Reply:
Sounds like pyright being dumb (what else is new) but you should change the type ignore to pyright ignore report MessageTypeHere or whatever the syntax is

Reply:
`render_mode` currently doesn't seem to exist in `AECEnv`, so there's something iffy going on here

Reply:
> Instead of ignoring type issues, add explicit assertions

We should probably do a pr one day fixing the ignored type errors to be assertions cause I know there’s a good amount